File: CargaClic-SPA/src/app/_models/Inventario/inventariogeneral.ts
export interface InventarioGeneral {
     id:  number ;
     lodNum:  string ;
     lodId: number;
     productoId: any ;
     descripcionLarga:  string ;
     ubicacionId:  number ;
     ubicacion:  string ;
     lotNum:  string ;
     fechaExpire:  Date ;
     untQty:  number ;
     untPak:  number ;
     fechaRegistro:  Date ;
     fechaUltMovimiento: Date;
     usuarioIngreso: number;
     ubicacionIdUlt: number;
     ubicacionUltima: string;
     huellaId: number;
     codigoHuella: string;
}

File: Domain/CargaClic.Domain/Inventario/InventarioGeneral.cs
using System;
using CargaClic.Common;

namespace CargaClic.Domain.Inventario
{
    public class InventarioGeneral : Entity
    {
        public long Id { get;set; }
        public long LodId {get;set;}
        public Guid ProductoId { get; set; }
        public string LotNum { get; set; }
        public DateTime? FechaExpire { get; set; }
        public DateTime? FechaManufactura { get; set; }
        public int UntQty { get; set; }
        public int? UntCas { get; set; }
        public int? UntPak { get; set; }
        public DateTime FechaRegistro { get; set; }
        public DateTime? FechaUltMovimiento { get; set; }
        public int UsuarioIngreso { get; set; }
        public int HuellaId { get; set; }
        public long? LineaId {get;set;}
        public Guid? OrdenReciboId {get;set;}
        public bool? Almacenado {get;set;}
        public int EstadoId {get;set;}
        public int ClienteId {get;set;}

    }
}